### DISCUSSION TITLE
Question: Can I variable a field from query A and use the output for Query B?

### DISCUSSION THREAD

**Discussion by JustNZ** [grafana/grafana-json-datasource#190] [Q&A]
Hello,
I have two query's (A & B) and I need the return from query A to use it in query B.
Is that possible for this json datasource?

![image](https://user-images.githubusercontent.com/51332147/137705948-b445438f-f38f-46a0-9a7e-e054e0fde208.png)

**Comment by marcusolsson** [comment]
Unfortunately this isn't possible, but it would be really cool if it did support this at some point. It would be possible to add a drop-down to define a parent query, but I would need to figure out how to refer to the intermediate result somehow. With JSONata, you can define your own variables, but not sure how to do it with JSONPath.
